static-analysis.datadog.yml

```diff
@@ -1,9 +1,12 @@
 rulesets:
   - python-best-practices:
     rules:
-      nested-blocks:
+      comment-fixme-todo-ownership:
         ignore:
           - "**"
       max-function-lines:
-        ignore: 
+        ignore:
           - "tests/ci_visibility/api/fake_runner_*.py"
+      nested-blocks:
+        ignore:
+          - "**"
```

Comment:
Alphabetized because I want to be a good citizen.

Reply:
![](https://as2.ftcdn.net/v2/jpg/00/35/38/77/1000_F_35387705_UxdOlcXyhE2y81W9DRg6VOWPZUxIe0Tz.jpg)